src/algebra/symmetrized.lean

```diff
@@ -0,0 +1,205 @@
+/-
+Copyright (c) 2021 Christopher Hoskin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christopher Hoskin
+-/
+import algebra.module.basic
+import tactic.abel
+
+/-!
+# Symmetrized algebra
+
+A commutative multiplication on a real or complex space can be constructed from any multiplication
+by "symmetrization" i.e
+$$
+a \circ b = \frac{1}{2}(ab + ba)
+$$
+
+We provide the symmetrized version of a type `α` as `sym_alg α`, with notation `αˢʸᵐ`.
+## Implementation notes
```

Comment:
```suggestion
We provide the symmetrized version of a type `α` as `sym_alg α`, with notation `αˢʸᵐ`.

## Implementation notes
```
Looks like that didn't apply as I intended - needs a blank line

Reply:
Thanks, yes I spotted that.